core/src/main/java/io/grpc/internal/ManagedClientTransport.java

```diff
@@ -110,6 +110,8 @@ interface Listener {
      * Called just before {@link #transportReady} to allow direct modification of transport
      * Attributes.
      */
-    Attributes filterTransport(Attributes attributes);
+    default Attributes filterTransport(Attributes attributes) {
+      return attributes;
```

Comment:
Minor: just for the sake of code coverage numbers, let's add a simple unit test.

Reply:
Done.